app/views/PillarAnalysis/AnalyticalStatementInput/AnalyticalEntryInput/index.tsx

```diff
@@ -108,7 +108,8 @@ function AnalyticalEntryInput(props: AnalyticalEntryInputProps) {
     const { entries } = useContext(EntryContext);
     const entry = value.entry ? entries[value.entry] : undefined;
 
-    const authorName = entry?.lead.authors?.[0]?.shortName;
+    const authors = entry?.lead.authors
+        ?.map((author) => (author.shortName)).join(', ');
```

Comment:
the `authors` property might have `mergedAs` property that has the real author value.